src/streamlink_cli/argparser.py

```diff
@@ -1116,6 +1116,14 @@ def build_parser():
         when using copyts.
         """
     )
+    transport.add_argument(
+        "--ffmpeg-copyts",
+        action="store_true",
+        help="""
+        Do not remove the initial start time offset value.
+        ffmpeg-start-at-zero option is set to "false".
+        """
```

Comment:
```suggestion
        help="""
        Forces the -copyts ffmpeg option and does not remove
        the initial start time offset value.

        --ffmpeg-no-start-at-zero will be set to True.
        """
```

The CLI docs should only reference other CLI arguments and not session option names, as it's not clear to the user what exactly will be done. The argument should also ideally be defined before `--ffmpeg-no-start-at-zero` and adding a double-dash will make the doc's argparser turn the referred arg into a link.